source/agora/test/NetworkManager.d

```diff
@@ -76,16 +77,22 @@ unittest
 
             Block last_block;
             // make 20 blocks which have an invalid previous hash
-            foreach (idx; 0 .. 20)
+
```

Comment:
I am not sure if this is the correct change. This code segment created invalid transactions **on purpose**, to test the behavior of the system. I will have to double-check the code again tomorrow to see if it does the right thing here.

Reply:
It was not the correct change because it was not creating invalid transactions. I left it untouched in the new commit.
`TxBuilder` does not allow one to build invalid transactions, therefore, it would have to be modified after it has been built.
I will try this in a separate issue.